cmd/ubuntu-image/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/canonical/ubuntu-image/commands"
+	"github.com/jessevdk/go-flags"
+)
+
+var osExit = os.Exit
+
+var stateMachineLongDesc = `Options for controlling the internal state machine.
+Other than -w, these options are mutually exclusive. When -u or -t is given,
+the state machine can be resumed later with -r, but -w must be given in that
+case since the state is saved in a .ubuntu-image.pck file in the working directory.`
+
+
+func main() {
+
+	parser := flags.NewParser(&commands.UbuntuImageCommand, flags.Default)
+	parser.AddGroup("[State Machine Options]", stateMachineLongDesc, &commands.StateMachineOpts)
+	parser.AddGroup("[Common Options]", "Options common to both commands", &commands.CommonOpts)
```

Comment:
Stylistic thing: I'm not sure if I'm fond of the `[<SECTION>]` format. If you look at the help output for other sections added automatically like `Help Options`, `Available Commands` etc. those are without the square brackets. Can we just leave those out?

Reply:
Sure that's an easy change.